packages/docs/src/components/navigator.tsx

```diff
@@ -156,7 +184,7 @@ export class Navigator extends React.PureComponent<INavigatorProps, INavigatorSt
                 </a>,
             ];
         }
-        return <Menu>{items}</Menu>;
+        return <div className={Classes.MENU} ref={this.refHandlers.menu}>{items}</div>;
```

Comment:
switching to `div` so `ref` will give me a DOM element, not a React instance, so i can `querySelector` above